src/yuzu/main.cpp

```diff
@@ -2285,8 +2360,16 @@ void GMainWindow::UpdateUITheme() {
     QStringList theme_paths(default_theme_paths);
 
     if (is_default_theme || current_theme.isEmpty()) {
-        qApp->setStyleSheet({});
-        setStyleSheet({});
+        const QString theme_uri(QLatin1Char{':'} + QStringLiteral("default/style.qss"));
```

Comment:
Any reason this isn't:

```suggestion
        const QString theme_uri(QStringLiteral(":default/style.qss"));
```